operator/src/main/resources/scripts/stop-server.py

```diff
@@ -24,6 +24,13 @@ def getEnvVar(var):
 domain_name = getEnvVar('DOMAIN_NAME')
 domain_path = getEnvVar('DOMAIN_HOME')
 service_name = getEnvVar('SERVICE_NAME')
+local_admin_port = sys.argv[1]
+local_admin_protocol = sys.argv[2]
+timeout = sys.argv[3]
+ignore_sessions = sys.argv[4]
+force = sys.argv[5]
+
```

Comment:
Could these be passed as env vars?  IMO, it's confusing to pass values in both as env vars and as command line args - and now this script is doing both, plus env vars are easier to maintain over time...